Dockerfiles/midstream/unit_tests.py

```diff
@@ -0,0 +1,41 @@
+
+import unittest
+import os
+import subprocess
+
+
+class Testing(unittest.TestCase):
+
+    def test_postive(self):
+        exec_cmd = """
+        /run_tests.py
+        """
+        result = subprocess.run(exec_cmd, shell=True)
+        self.assertEqual(0, result.returncode)
+
+    def test_negative_extract_operator_bundle(self):
+        exec_cmd = """
+        export UMOCI_BIN_PATH=/root
```

Comment:
Please add commemnt to the function what do you try to test here.
It is not clear to me why we should fail with return code 55 for invalid umoci path